src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -58,7 +58,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\CodePageNameMappings.csv" />
```

Comment:
[Doesn't look like it](http://stackoverflow.com/questions/1060870/in-a-csproj-file-what-is-is-none-include-for).